src/pages/guides/how-to/index.md

```diff
@@ -7,9 +7,11 @@ description: How to use AEM OpenAPI-based APIs
 
 ### Introduction
 
-AEM as a Cloud Service offers a growing list of APIs that adhere to the [OpenAPI Specification](https://swagger.io/specification/v2/) (version 2), including operations on [Content Fragments](../../api/experimental/sites/) resources.
+AEM as a Cloud Service offers a variety of APIs that adhere to the [OpenAPI Specification](https://swagger.io/specification/v2/) (version 2).
```

Comment:
The new APIs are OpenAPI 3.0.3. Should we rather link to https://spec.openapis.org/oas/v3.0.3?

Reply:
Good point, I changed the link